rpc/wrtc_signaling_answerer.go

```diff
@@ -478,7 +478,12 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		return err
 	}
 	if err := sendDone(); err != nil {
```

Comment:
should we be more conservative and only warn if there is an EOF? or are we pretty comfortable just warning on any error at this point?

Reply:
I think the more "conservative" behavior is to ignore all errors. As per the conversion [following this](https://viaminc.slack.com/archives/C06542APFRQ/p1707832372930119?thread_ts=1707763492.626749&cid=C06542APFRQ). The error handling here is an optimization that we demonstrate goes wrong. It's more conservative to not have that optimization.

I don't see any reason why the more conservative change* of only suppressing EOF has an advantage.

Reply:
cool, thanks for the discussion link! makes sense, I'm on board.